justfile

```diff
@@ -3,18 +3,37 @@
 #   like a script, with `./justfile test`, for example.
 
 set shell := ["bash", "-c"]
+set positional-arguments := true
 
 export NIGHTLY_TOOLCHAIN := "nightly-2023-04-18"
 export CARGO_TERM_COLOR := "always"
 export RUST_BACKTRACE := "1"
 
+default:
+  @just --justfile {{ justfile() }} --list
+
 nightly-toolchain:
   rustup install $NIGHTLY_TOOLCHAIN
   rustup component add rust-src --toolchain $NIGHTLY_TOOLCHAIN
 
 install-clippy:
   rustup component add clippy
 
+# run clippy on all workspace members
+check-workspace: install-clippy
+  cargo clippy
+
+# run clippy on a vendored crate (e.g. libressl-src)
+check-crate NAME FEATURES: install-clippy
+  #!/usr/bin/env bash
+  cleanup() {
+    find {{ justfile_directory() / "crates" }} -name Cargo.lock -exec rm -f '{}' +
+  }
+  trap cleanup EXIT
```

Comment:
I think this is a little dangerous to have in a file that can be executed by users. I maybe don't want to remove my lock file when running a check locally.